docs/digger/installation/digger-install-intro.asciidoc

```diff
@@ -1,17 +1,23 @@
-== Digger Installation Introduction
+== AeroGear Digger Installation
 //Please DO NOT use 'Overview' as a section heading anywhere http://stylepedia.net/#sect-Red_Hat_Technical_Publications-Writing_Style_Guide-Overall_Book_Design-Unused_Heading_Titles
-GitHub repo with Ansible job used for Digger installation https://github.com/aerogear/digger-installer[Digger Installer tool^].
+https://github.com/aerogear/digger-installer[Digger Installer^] - GitHub repo with Ansible playbook used for Digger installation.
 
-The AeroGear Digger project allows you to setup a service that has the capability to build mobile applications using OpenShift containers.
+The AeroGear Digger project allows you to setup a CI/CD server that is able to build mobile applications using OpenShift containers.
 
-It is made of following components:
+NOTE: Digger can be installed behind HTTP/HTTPS proxy that do not require authentication.
```

Comment:
IMHO we don't need this note here.

Reply:
i would like to mention it in the intro, this is not the best way to do it

Reply:
resolved